src/styles/buttons.css

```diff
@@ -117,6 +117,11 @@ button,
   --border-hover: var(--orange-hover);
 }
 
+.btn--accent {
+  --color: var(--green);
+  --color-hover: var(--green-hover);
+}
```

Comment:
Accent color should depend on the `--accent` variable as it does in the [`.c-accent` class](https://github.com/santiment/san-webkit/blob/master/src/styles/text.css#L94-L96)